src/lib/cli/run.command.ts

```diff
@@ -14,10 +14,11 @@ export class RunCommand implements CliCommand {
         include: ['**/*.spec.ts']
     };
 
-    constructor(private logger: Logger, private _configFile: string = 'testy.json') { }
+    constructor(private logger: Logger, private _configFile: string = 'testy.json', private _tsConfigFile = 'tsconfig.json') { }
```

Comment:
Any reason in particular for prefixing these two with underscore? (and not logger)

Reply:
They are prefixed with underscores because they have public getters, whereas the logger is purely private. The public getters are currently only used for testing purposes though, which is not ideal IMO.